.github/workflows/tests.yml

```diff
@@ -16,11 +16,20 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: install fast_float dependency
+      run: |
+        cd ${{github.workspace}}
+        git clone https://github.com/fastfloat/fast_float.git
+        mkdir fast_float/build
+        cd fast_float/build
+        cmake -DCMAKE_INSTALL_PREFIX="${{github.workspace}}/installation" ..
+        make install
+
     - name: Configure CMake
       run: |
         mkdir build
         cd build
-        cmake -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} ..
+        cmake -DFastFloat_DIR="${{github.workspace}}/installation/share/cmake/FastFloat" -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} ..
```

Comment:
see comments about the libraries. Also do we need this in the action if it is in the cmake install?

Reply:
hmm I forgott, maybe sudo make install would work above. Because I did not have the rights to install therefore I changed the prefix and cmake will not search in this path. I will try if it is working